cypress/component/Auth/auth.spec.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-undef */
+
+import {Auth} from '../../../src/libs/auth/auth';
+import {Config} from '../../../src/libs/config';
+import {GoogleIS} from '../../../src/libs/googleIS';
+import {OAuth2} from '../../../src/libs/ajax/OAuth2';
+import {Storage} from '../../../src/libs/storage';
+
+describe('Auth', function () {
+  // Intercept configuration calls
+  beforeEach(async () => {
+    cy.intercept({
+      method: 'GET',
+      url: '/config.json',
+      hostname: 'localhost',
+    }, {'env': 'ci'});
+    cy.stub(OAuth2, 'getConfig').returns({
+      'authorityEndpoint': 'authorityEndpoint',
+      'clientId': 'clientId'
+    });
+    await Auth.initialize();
+  });
+  it('Sign Out Clears the session when called', async function () {
```

Comment:
Do you store anything in localStorage on DUOS? That might also be good to check if so.

Reply:
We do - I'll boost the uses of `Storage.setXYZ()` and check those too, thank you!